1.26/Dockerfile.rhel9

```diff
@@ -39,7 +39,7 @@ ENV NGINX_CONFIGURATION_PATH=${APP_ROOT}/etc/nginx.d \
     NGINX_PERL_MODULE_PATH=${APP_ROOT}/etc/perl
 
 RUN yum -y module enable nginx:$NGINX_VERSION && \
-    INSTALL_PKGS="nss_wrapper-libs bind-utils gettext hostname nginx nginx-mod-stream nginx-mod-http-perl" && \
+    INSTALL_PKGS="nss_wrapper-libs bind9.18-utils gettext hostname nginx nginx-mod-stream nginx-mod-http-perl" && \
```

Comment:
Can we change this to `nginx-all-modules` rather than just `nginx-mod-stream nginx-mod-http-perl`?

Reply:
Sure, we can use it. I will do it., Thanks.

Reply:
Fixed by https://github.com/sclorg/nginx-container/pull/326/commits/138be4063798ad28f35120344c193e3a73d6fd48